src/seq/nucleotide.jl

```diff
@@ -103,6 +103,24 @@ function isGC(nt::DNANucleotide)
 end
 
 """
+    isPurine(nt::Nucleotide)
+
+Test if nucleotide is surely a Purine.
+"""
+function isPurine(nt::Nucleotide)
+    return (compatbits(nt) | 0b0101) == 0b0101
```

Comment:
This is wrong. When `nt = DNA_Gap` `compatbits(nt)` is `0b0000` and hence `(compatbits(nt) | 0b0101) = 0b0101`. And the result is `isPurine(DNA_Gap)` is `true`.